src/main/java/com/google/devtools/build/lib/worker/Worker.java

```diff
@@ -41,13 +41,13 @@
  * class.
  */
 class Worker {
-  private final WorkerKey workerKey;
-  private final int workerId;
-  private final Path workDir;
-  private final Path logFile;
+  protected final WorkerKey workerKey;
```

Comment:
Given that you probably went through the hassle of figuring out what all these mean, I'd appreciate adding docstrings. (Which are required anyway because turning these into protected makes them effectively public.)

Reply:
Docstrings added. These are based on my understanding, they could be wrong.